app/var/projects.json

```diff
@@ -1909,6 +1909,34 @@
         "phalcon 3x",
         "responsive design"
       ]
+    },
+    {
+      "permalink": "learny-online",
+      "name": "Learny Online",
+      "summary": "Learny Online is an online educational platform with exclusive, original and quality content.",
+      "description": "A learny is a course on demand that combines videos and written content of high quality, accompanied by a test that will allow you to develop skills and acquire tools without the need to spend time and spend money to attend face-to-face classes. You only have to be connected to the Internet from your mobile, tablet or computer and start learning in a pleasant and simple way, at the moment you decide.",
+      "url": "https://learny.online",
+      "info": "https://learny.online/paginas-informativas",
+      "src": "",
+      "submitter": "emiliodeg",
+      "submissionDate": "2018-02-21",
+      "images": [
+        "https://learny.online/assets/images/logo/learny-online.png"
```

Comment:
This will not work. The image(s) need to be part of the PR. One has to be the thumb image and they have to be located in the relevant folder which is your `permalink` above

Reply:
ok I didn't know, don't worry I will fix it this nigth